### ISSUE TITLE
Rights statement should define access permissions

### ISSUE THREAD

**Issue by terrellt** [OregonDigital/oregondigital_2#177]

**Comment by jsimic** [comment]
So this will no longer be two different metadata fields?

**Comment by terrellt** [comment]
The current recommendation from the Hydra rights sub-group is here: https://wiki.duraspace.org/display/hydra/Rights+Metadata+Subgroup

Effectively use EDM.rights for URIs, rightsHolder for things like RR-R, and DCE.rights for human readable rights information.

**Comment by jsimic** [comment]
@srabun Just reading your reply to Randy Parks...

**Comment by wickr** [comment]
@terrellt wouldn't RR-R also go in EDM.rights? I read it as rightsHolder would be URIs for institutions or other entities, maybe people

**Comment by kestlund** [comment]
@wickr  is correct.

**Comment by terrellt** [comment]
@wickr Yes- - I meant the rightsHolder would be used to figure out access permissions in cases where rights are restricted.

**Comment by jsimic** [comment]
Some collections contain licensed public domain permissions, therefore we created accessPermissions field to prevent this material from appearing to non-institutional users.

**Comment by jechols** [comment]
Just a quick note from a discussion today: we need to make sure we don't give blanket rights just because somebody is a curator.  e.g., being allowed to ingest an Image for OSU doesn't mean you can download restricted images from a UO collection.
